#imu.msg

float32 accel_x  # linear acceleration
float32 accel_y  # linear acceleration
float32 accel_z  # linear acceleration
float32 gyro_x   # angular velocity
float32 gyro_y   # angular velocity
float32 gyro_z   # angular velocity